---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

<!-- {% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %} -->


**期刊论文**  
  
[1] Zhou Q, Fu H, Yang D, Ye C, Zhu S, Lin J, Ye W, Ji G, Ye X, Wu X*, Li QQ*: Differential alternative polyadenylation contributes to the developmental divergence between two rice subspecies Japonica and Indica. The Plant Journal 2019, 0(ja). (JCR2#, Top)  
  
[2] Ye W, Long Y, Ji G, Su Y, Ye P, Fu H, Wu X*: Cluster analysis of replicated alternative polyadenylation data using canonical correlation analysis. BMC Genomics 2019, 20(75):1-15. (JCR2#)  
  
[3] Ye C, Long Y, Ji G, Li QQ*, Wu X*: APAtrap: identification and quantification of alternative polyadenylation sites from RNA-seq data. Bioinformatics 2018, 34(11):1841-1849. (JCR2#, Top)  
  
[4] Ji G, Ye W, Su Y, Chen M, Huang G, Wu X*: AStrap: identification of alternative splicing from transcript sequences without a reference genome. Bioinformatics 2018:bty1008-bty1008. (JCR2#, Top)
  
[5] Ji G, Chen M, Ye W, Zhu S, Ye C, Su Y, Peng H, Wu X*: TSAPA: identification of tissue-specific alternative polyadenylation sites in plants. Bioinformatics 2018, 34(12):2123-2125. (JCR2#, Top) 
  
[6] Ji G, Lin Q, Long Y, Ye C, Ye W, Wu X*: PAcluster: Clustering polyadenylation site data using canonical correlation analysis. J Bioinf Comput Biol 2017, 15(5):19. (JCR4#)    

[7] Fu H, Yang D, Su W, Ma L, Shen Y, Ji G, Ye X*, Wu X*, Li QQ*: Genome-wide dynamics of alternative polyadenylation in rice. Genome Res 2016, 26(12):1753-1760. (JCR1#, Top)    

[8] Wu, X.*, Zhang, Y., and Li, Q. Q.* (2016) PlantAPA: a portal for visualization and analysis of alternative polyadenylation in plants. Frontiers in Plant Science 7 (JCR2#)    

[9] Wu, X.*, Zeng, Y., Guan, J., Ji, G., Huang, R., and Li, Q. Q.* (2015) Genome-wide characterization of intergenic polyadenylation sites redefines gene spaces in Arabidopsis thaliana. BMC Genomics 16, 015-1691 (JCR2#)    

[10] Ji, G., Li, L., Li, Q. Q., Wu, X., Fu, J., Chen, G., and Wu, X.* (2015) PASPA: a web server for mRNA poly(A) site predictions in plants and algae. Bioinformatics 31, 1671-1673 (JCR2#, Top)  
  
[11] Ji, G., Huang, G., Yang, Z., Wu, X., Chen, X., and Yuan, M. (2015) Using consensus interval partial least square in near infrared spectra analysis. Chemometrics Intellig. Lab. Syst. 144, 56-62 (JCR3#) 
  
[12] Ji, G., Guan, J., Zeng, Y., Li, Q. Q., and Wu, X.* (2015) Genome-wide identification and predictive modeling of polyadenylation sites in eukaryotes. Briefings in Bioinformatics 16, 304-313 (JCR1#, Top)  
  
[13] Zhao, Z., Wu, X., Raj Kumar, P. K., Dong, M., Ji, G., Li, Q. Q., and Liang, C. (2014) Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms. G3: Genes|Genomes|Genetics 4, 871-883 (JCR3#)  
  
[14] Wu, X., Gaffney, B., Hunt, A., and Li, Q. (2014) Genome-wide determination of poly(A) sites in Medicago truncatula: evolutionary conservation of alternative poly(A) site choice. BMC Genomics 15, 615 (JCR2#)  
  
[15] Wu, X.*, Ji, G., and Zeng, Y. (2012) In silico prediction of mRNA poly(A) sites in Chlamydomonas reinhardtii. Mol. Genet. Genomics 287, 895-907 (JCR3#)  
  
[16] Thomas, P. E., Wu, X., Liu, M., Gaffney, B., Ji, G., Li, Q. Q., and Hunt, A. G. (2012) Genome-Wide Control of Polyadenylation Site Choice by CPSF30 in Arabidopsis. Plant Cell 24, 4376-4388 (JCR1#, Top)  
  
[17] Li, H. J., Ye, C. T., Ji, G. L., Wu, X. H., Xiang, Z., Li, Y. Y., Cao, Y. H., Liu, X. L., Douek, D. C., Price, D. A., and Han, J. H. (2012) Recombinatorial Biases and Convergent Recombination Determine Interindividual TCR beta Sharing in Murine Thymocytes. J. Immunol. 189, 2404-2413 (JCR2#)  
  
[18] Zheng, J., Xing, D., Wu, X., Shen, Y., Kroll, D. M., Ji, G., and Li, Q. Q. (2011) Ratio-based analysis of differential mRNA processing and expression of a polyadenylation factor mutant pcfs4 using Arabidopsis Tiling microarray. PLoS ONE 6, 10 (JCR3#)  
  
[19] Wu, X., Liu, M., Downie, B., Liang, C., Ji, G., Li, Q. Q., and Hunt, A. G. (2011) Genome-wide landscape of polyadenylation in Arabidopsis provides evidence for extensive alternative polyadenylation. Proc. Natl. Acad. Sci. USA 108, 12533-12538 (JCR1#, Top)  
  
[20] Shen, Y., Venu, R. C., Nobuta, K., Wu, X., Notibala, V., Demirci, C., Meyers, B. C., Wang, G.-L., Ji, G., and Li, Q. Q. (2011) Transcriptome dynamics through alternative polyadenylation in developmental and environmental responses in plants revealed by deep sequencing. Genome Res. 21, 1478-1486 (JCR1#, Top)  
  
[21] Ji, G., Wu, X., Shen, Y., Huang, J., and Li, Q. Q. (2010) A classification-based prediction model of messenger RNA polyadenylation sites. J. Theor. Biol. 265, 287-296 (JCR3#)  
  
[22] Shen, Y., Ji, G., Haas, B. J., Wu, X., Zheng, J., Reese, G. J., and Li, Q. Q. (2008) Genome level analysis of rice mRNA 3'-end processing signals and alternative polyadenylation. Nucleic Acids Res. 36, 3150-3161 (JCR1#, Top)  
  
[23] Ji, G., Zheng, J., Shen, Y., Wu, X., Jiang, R., Lin, Y., Loke, J. C., Davis, K. M., Reese, G. J., and Li, Q. Q. (2007) Predictive modeling of plant messenger RNA polyadenylation sites. BMC Bioinformatics 8, 43 (JCR3#)  
  

**论著章节**  
  
[1]Wu, X., Ji, G., and Li, Q. Q. (2015) Poly(A)-Tag Deep Sequencing Data Processing to Extract Poly(A) Sites. In Polyadenylation in Plants : Methods and Protocols Vol. 1255 pp. 39-48, Springer New York, New York  
  
[2]Wu, X., Ji, G., and Li, Q. Q. (2015) Prediction of Plant mRNA Polyadenylation Sites. In Polyadenylation in Plants : Methods and Protocols Vol. 1255 pp. 13-23  
  
[3]Wu, X., Ji, G., and Li, Q. Q. (2015) Computational Analysis of Plant Polyadenylation Signals. In Polyadenylation in Plants : Methods and Protocols Vol. 1255 pp. 3-11  
  
[4]Liu, M., Wu, X., and Li, Q. Q. (2015) DNA/RNA Hybrid Primer Mediated Poly(A) Tag Library Construction for Illumina Sequencing. In Polyadenylation in Plants : Methods and Protocols Vol. 1255 pp. 175-184, Springer New York  
  

**会议论文**  
  
[1]Wang, Y., Sun, C., Hong, H., and Wu, X. * (2015) Genome-wide analysis of the associations between polyadenylation sites and repeated sequences in Arabidopsis thaliana. In Biomedical Engineering and Informatics (BMEI), 2015 8th International Conference on pp. 537-541  
  
[2]Wu, X., Tang, M., Yao, J., Lin, S., Xiang, Z., and Ji, G. (2012) PATMAP: Polyadenylation Site Identification from Next-Generation Sequencing Data. In Hybrid Artificial Intelligent Systems Vol. 7208 pp. 485-496, Springer Berlin / Heidelberg, Salamanca  
  
[3]Ji, G., Xiang, Z., Wu, X., Lin, S., Tang, M., and Yao, J. (2012) High-throughput antibody sequence alignment based on GPU computing. In 27th Annual ACM Symposium on Applied Computing, SAC 2012, March 26, 2012 - March 30, 2012 pp. 1417-1418, Association for Computing Machinery, Trento, Italy 
  
[4]Wu, X., Liu, Q., Tang, M., Zhang, H., Yao, J., and Ji, G. (2011) Identification of mRNA poly(A) signal patterns. In 2011 International Symposium on Innovations in Intelligent Systems and Applications (INISTA 2011)  
  
[5]Ji, G., Wu, X., Li, Q., and Zheng, J. (2010) Messenger RNA polyadenylation site recognition in green alga Chlamydomonas Reinhardtii. Lecture Notes in Computer Science 6063, 17-26  
  
[6]Ji, G., Wu, X., Xing, D., and Li, Q. Q. (2009) PROBER: segmentation and differential analysis tool for tiling microarray data. In Proceedings of the 2009 2nd International Conference on Biomedical Engineering and Informatics (BMEI) p. 5 pp. 
  
