---
layout: archive
title: "学术论文"
permalink: /publications/
author_profile: true
---

<!-- {% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %} -->

# 代表性论文（第一或通讯作者）

## 2022

- Ye W, Lian Q, Ye C, **Wu X\***
  A survey on computational approaches for predicting polyadenylation sites from DNA sequences, bulk RNA-seq and single-cell RNA-seq
  **Genomics, Proteomics & Bioinformatics**
  [FullText](https://authors.elsevier.com/sd/article/S1672022922001218)
- Liu M, Hao L, Yang S, **Wu X\***
  PolyAtailor: measuring poly(A) tail length from short-read and long-read sequencing data
  **Briefings in Bioinformatics**
  [FullText](https://doi.org/10.1093/bib/bbac271) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35769001/) | [Tool](https://github.com/BMILAB/PolyAtailor)
- Leote AC#, **Wu X#**, Beyer A\*
  Regulatory network-based imputation of dropouts in single-cell RNA sequencing data
  **PLOS Computational Biology**
  [FullText](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009849) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35176023/) | [Tool](https://bioconductor.org/packages/release/bioc/html/ADImpute.html)

## 2021

- Zhu S, Lian Q, Ye W, Qin W, Wu Z, Ji G, **Wu X\***
  scAPAdb: a comprehensive database of alternative polyadenylation at single-cell resolution
  **Nucleic Acids Research**
  [FullText](https://academic.oup.com/nar/article/50/D1/D365/6368523) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34508354/) | [Database](http://www.bmibig.cn/scAPAdb/)
- **Wu X\***, Liu T, Ye C, Ye W, Ji G
  scAPAtrap: identification and quantification of alternative polyadenylation sites from single-cell RNA-seq data
  **Briefings in Bioinformatics**
  [FullText](https://academic.oup.com/bib/article/22/4/bbaa273/5952304) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33142319/) | [Tool](https://github.com/BMILAB/scAPAtrap)
- Ye W#, Liu T#, Fu H, Ye C, Ji G\*, **Wu X\***
  movAPA: Modeling and visualization of dynamics of alternative polyadenylation across biological samples
  **Bioinformatics**
  [FullText](https://doi.org/10.1093/bioinformatics/btaa997) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33258917/) | [Tool](https://github.com/BMILAB/movAPA)

## 2020

- Chen M, Ji G, Fu H, Lin Q, Ye C, Ye W, Su Y, **Wu X\***
  A survey on identification and quantification of alternative polyadenylation sites from RNA-seq data
  **Briefings in Bioinformatics**
  [FullText](https://doi.org/10.1093/bib/bbz068) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31267126/) | [Code](https://github.com/BMILAB/APAsurvey)
- Ye P, Ye W, Ye C, Li S, Ye L, Ji G, **Wu X\***
  scHinter: Imputing dropout events for single-cell RNA-seq data with limited sample size
  **Bioinformatics**
  [FullText](https://doi.org/10.1093/bioinformatics/btz627) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31392316/) | [Tool](https://github.com/BMILAB/scHinter)
- Zhu S#, Ye W#, Ye L, Fu H, Ye C, Xiao X, Ji Y, Lin W, Ji G, **Wu X\***
  PlantAPAdb: A Comprehensive Database for Alternative Polyadenylation Sites in Plants
  **Plant Physiology**
  [FullText](https://doi.org/10.1104/pp.19.00943) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31767692/) | [Database](http://www.bmibig.cn/plantAPAdb/)
- Ye W#, Wang T#, Wei W, Lou S, Lan F, Zhu S, Li Q, Ji G, Lin C, **Wu X\***, Ma L\*
  The full-length transcriptome of Spartina alterniflora reveals the complexity of high salt tolerance in monocotyledonous halophyte
  **Plant and Cell Physiology**
  [FullText](https://doi.org/10.1093/pcp/pcaa013)  | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32044993/) | [Database](http://plantpolya.org/SAPacBio/)

## 2019

-  Ji G, Ye W, Su Y, Chen M, Huang G, **Wu X\***
  AStrap: identification of alternative splicing from transcript sequences without a reference genome
  **Bioinformatics **
  [FullText](https://doi.org/10.1093/bioinformatics/bty1008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30535139/) | [Tool](https://github.com/BMILAB/AStrap)
- Zhou Q#, Fu H#, Yang D, Ye C, Zhu S, Lin J, Ye W, Ji G, Ye X, **Wu X\***, Li QQ\*
  Differential alternative polyadenylation contributes to the developmental divergence between two rice subspecies Japonica and Indica
  **The Plant Journal**
  [FullText](https://onlinelibrary.wiley.com/doi/10.1111/tpj.14209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30570805/) 

## 2018

- Ji G, Chen M, Ye W, Zhu S, Ye C, Su Y, Peng H, **Wu X\***
  TSAPA: identification of tissue-specific alternative polyadenylation sites in plants
  **Bioinformatics**
  [FullText](https://doi.org/10.1093/bioinformatics/bty044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385403/) | [Tool](https://github.com/BMILAB/TSAPA)
- Ye C, Long Y, Ji G, Li QQ\*, **Wu X\***
  APAtrap: identification and quantification of alternative polyadenylation sites from RNA-seq data
  **Bioinformatics**
  [FullText](https://doi.org/10.1093/bioinformatics/bty029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29360928/) | [Tool](https://apatrap.sourceforge.io)

## Before 2018

- Fu H, Yang D, Su W, Ma L, Shen Y, Ji G, Ye X, **Wu X\***, Li QQ\*
  Genome-wide dynamics of alternative polyadenylation in rice
  **Genome Research, 2016**
  [FullText](https://genome.cshlp.org/content/26/12/1753) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27733415/) 

- Ji G, Guan J, Zeng Y, Li QQ, **Wu X\***
  Genome-wide identification and predictive modeling of polyadenylation sites in eukaryotes
  **Briefings in Bioinformatics, 2015**
  [FullText](https://doi.org/10.1093/bib/bbu011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24695098/) 

- Ji G, Li L, Li QQ, Wu X, Fu J, Chen G, **Wu X\***
  PASPA: a web server for mRNA poly(A) site predictions in plants and algae
  **Bioinformatics, 2015**
  [FullText](https://doi.org/10.1093/bioinformatics/btv004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25583118/) | [Tool](http://www.bmibig.cn/mnt/tools/paspa/)

- **Wu X**, Liu M, Downie B, Liang C, Ji G, Li QQ\*, Hunt AG\*
  Genome-wide landscape of polyadenylation in Arabidopsis provides evidence for extensive alternative polyadenylation
  **Proc Natl Acad Sci USA, 2011**
  [FullText](https://www.pnas.org/doi/10.1073/pnas.1019732108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21746925/)

  

# 代表性论文（合作作者）

- Ye C\*, Zhao D, Ye W, **Wu X**, Li Q, Lin J\*. 2021. QuantifyPoly(A): reshaping alternative polyadenylation landscapes of eukaryotes with weighted density peak clustering. **Briefings in Bioinformatics** bbab268.*
- Ye C\*#, Zhou Q\#, **Wu X**, Yu C, Ji G, Saban DR, Li QQ. 2019. scDAPA: detection and visualization of dynamic alternative polyadenylation from single cell RNA-seq data. **Bioinformatics** 36(4):1262-1264.
- Li H#, Ye C#, Ji G\*, **Wu X**, Xiang Z, Li Y, Cao Y, Liu X, Douek D, Price D, Han J\*. 2012. Recombinatorial Biases and Convergent Recombination Determine Interindividual TCR beta Sharing in Murine Thymocytes. **Journal of Immunol** 189(5):2404-2413. 
- Thomas PE, **Wu X**, Liu M, Gaffney B, Ji G, Li QQ\*, Hunt AG\*. 2012. Genome-Wide Control of Polyadenylation Site Choice by CPSF30 in Arabidopsis. **Plant Cell** 24(11):4376-4388.
- Fu H, Wang P, **Wu X**, Zhou X, Ji G, Shen Y, Gao Y, Li QQ\*, Liang J\*. 2019. Distinct genome-wide alternative polyadenylation during the response to silicon availability in the marine diatom Thalassiosira pseudonana. **Plant Journal** 99(10):67-80.
- Lin J, Xu R, **Wu X**, Shen Y, Li QQ\*. 2017. Role of Cleavage and Polyadenylation Specificity Factor 100: Anchoring Poly(A) Sites and Modulating Transcription Termination. **Plant Journal** 91:829-839.
- Hong L, Ye C, Lin J, Fu H, **Wu X**, Li QQ\*. 2017. Alternative polyadenylation is involved in auxin-based plant growth and development. **Plant Journal** 93(2):246-258.
- Shen Y, Venu RC, Nobuta K, **Wu X**, Notibala V, Demirci C, Meyers BC, Wang G-L, Ji G, Li QQ\*. 2011. Transcriptome dynamics through alternative polyadenylation in developmental and environmental responses in plants revealed by deep sequencing. **Genome Research** 21(9):1478-1486.
- Shen Y, Ji G, Haas BJ, **Wu X**, Zheng J, Reese GJ, Li QQ\*. 2008. Genome level analysis of rice mRNA 3'-end processing signals and alternative polyadenylation. **Nucleic Acids Research** 36(9):3150-3161.
